pilot/pkg/model/config.go

```diff
@@ -569,15 +591,22 @@ func ResolveShortnameToFQDN(host string, meta ConfigMeta) string {
 	return out
 }
 
+type authCache struct {
```

Comment:
May be name this jwksResolver or smt like that. Also, consider to move to separate file.

Reply:
jwksUriResolver ? as we're going to have another jwtPubKeyResolver